package.json

```diff
@@ -11,9 +11,14 @@
     "@craco/craco": "^6.0.0",
     "@okta/okta-react": "^3.0.2",
     "@okta/okta-signin-widget": "^4.1.2",
+    "@surma/rollup-plugin-off-main-thread": "^2.2.3",
```

Comment:
Can you please let me know what all these packages you've installed are being used for. That seems like a LOT of installations and this PR doesn't seem to be using any of them.

Reply:
I think these came up when I was having issues getting my local running and experimented with it a bit. I'll work on these changes now!